app/styles/components/_detailview.scss

```diff
@@ -1,13 +1,59 @@
 //course, session, etc detail view
 .detail-view {
   background-color: $background-grey;
-    border-style: solid;
+  border-style: solid;
   border-color: $header-grey;
   border-width: 0 2px 2px;
   border-top: 0px;
   margin-top: -4px;
     font-size: $base-font-size;
 
+  /* Because certain sections are styled differently from others (ie, some headers have togglesa and others do not, 
+     put the non-default, section type-specific styling here, put the defaults below */
+
+  /* session-offerings section */
+  .session-offerings {
+    section {
+      &.detail-block {
+
+        .detail-title {
+          @include span-columns(2 of 12);
+          @include omega();
+        }
+        .calendar-toggle {
+          @include span-columns(2 of 12);
+          @include omega();
+          margin-top: 1em;
+        }
+        .detail-actions{
+          @include shift(6.5 of 12);
```

Comment:
I think I was reading somewhere that it did, but it might have been something regarding bootstrap, so I'll try to find the article to make sure. But I do know it worked ALOT better when I added the .5 to the value...